voter-dapp/src/RetrieveRewards.js

```diff
@@ -14,13 +15,13 @@ function getOrCreateObj(containingObj, field) {
   return containingObj[field];
 }
 
-function useRetrieveRewardsTxn(retrievedRewardsEvents, revealedVoteEvents, priceResolvedEvents, votingAccount) {
+function useRetrieveRewardsTxn(retrievedRewardsEvents, revealedVoteEvents, priceRequests, votingAccount) {
   const { drizzle, useCacheSend } = drizzleReactHooks.useDrizzle();
   const { web3 } = drizzle;
 
   const { send, status } = useCacheSend("Voting", "retrieveRewards");
 
-  if (retrievedRewardsEvents === undefined || revealedVoteEvents === undefined || priceResolvedEvents === undefined) {
+  if (retrievedRewardsEvents === undefined || revealedVoteEvents === undefined || priceRequests == null) {
```

Comment:
missing third "="

Reply:
This was intentional so it would compare as equal to `null` or `undefined`. Is there a better way to signal that? Would `!priceRequests` be preferable?

Reply:
Changed to `!priceRequests`. PTAL.

Reply:
cool makes sense